sqlite_utils/cli.py

```diff
@@ -2701,6 +2714,120 @@ def convert(
             )
 
 
+@cli.command(
+    "add-geometry-column",
+    help="""Add a SpatiaLite geometry column to an existing table. Requires SpatiaLite extension.
```

Comment:
Any reason you used `help=` here rather than having this as a docstring on the comment? Asking because it's inconsistent with the other CLI functions in this module.